src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.Log.cs

```diff
@@ -76,8 +76,7 @@ public static void Log<T>(ReadOnlySpan<T> x, T y, Span<T> destination)
         internal readonly struct LogOperator<T> : IUnaryOperator<T, T>
             where T : ILogarithmicFunctions<T>
         {
-            public static bool Vectorizable => (typeof(T) == typeof(double))
-                                            || (typeof(T) == typeof(float));
+            public static bool Vectorizable => false; // typeof(T) == typeof(float) || typeof(T) == typeof(double); // TODO: https://github.com/dotnet/runtime/issues/100535
```

Comment:
What's failing for `Log`? It has a proper implementation and has tests covering all the same scenarios as scalar is testing?

The other APIs are the ones naively implemented in terms of log, and not actually checking for the relevant edge case handling.

Reply:
It's the overload of Log that takes a base.

Reply:
Thanks for confirming!